packages/slice-machine/types/cloudinary.d.ts

```diff
@@ -7,7 +7,11 @@ interface ImageProps extends JSX.IntrinsicElements.img {
   crop: string;
 }
 
-interface VideoProps extends JSX.IntrinsicElements.video {
+// interface VideoProps extends JSX.IntrinsicElements.video {
```

Comment:
Can you remove this ? We avoid commented line in the code : https://refactoring.guru/fr/smells/dead-code

Reply:
can do